Set the default value of "AWKPATH" and "AWKLIBPATH" to `.:$EXEDIR/awklib` where
`$EXEDIR` is the directory in which the GAWK binary resides.

--- main.c	2026-07-09 11:47:00.000000000 -0500
+++ main.c	2026-10-28 13:59:37.576554502 -0500
@@ -33,6 +33,8 @@
 #include <mcheck.h>
 #endif
 
+#include "../exepath.c"
+
 #define DEFAULT_PROFILE		"awkprof.out"	/* where to put profile */
 #define DEFAULT_VARFILE		"awkvars.out"	/* where to put vars */
 #define DEFAULT_PREC		53
@@ -218,10 +220,14 @@
 	bool have_srcfile = false;
 	SRCFILE *s;
 	char *cp;
+	static char path[PATH_MAX + 8] = ".:"; // 8 == strlen(".:awklib")
 #if defined(LOCALEDEBUG)
 	const char *initial_locale;
 #endif
 
+	if (executable_path(path + 2))
+		defpath = deflibpath = path, strcpy(rindex(path, '/') + 1, "awklib");
+
 	myname = gawk_name(argv[0]);
 
 	using_persistent_malloc = enable_pma(argv);
